core/string.c

```diff
@@ -174,40 +181,19 @@ d_r_memmove(void *dst, const void *src, size_t n)
  */
```

Comment:
Please update the comment which still talks about aliasing.  Explain why we can't alias.

Reply:
Missed that, thanks.